dis_snek/client/client.py

```diff
@@ -594,7 +584,7 @@ async def login(self, token, start_gateway=False) -> None:
         self._mention_reg = re.compile(rf"^(<@!?{self.user.id}*>\s)")
         self.dispatch(events.Login())
 
-    def start(self, token) -> None:
+    async def start(self, token) -> None:
```

Comment:
hmm.. Won't this means we cant do `bot.start(TOKEN)`?

Reply:
you have to do
```py
from asyncio import run
... # omitted stuff
run(bot.start(TOKEN))
```